lib/rucio/common/config.py

```diff
@@ -237,6 +237,72 @@ def config_get_bool(section, option, raise_exception=True, default=None, check_c
                 raise err_
 
 
+def config_get_list(section, option, raise_exception=True, default=None, check_config_table=True, session=None,
```

Comment:
Please rebase this and make it similar to how @joeldierkes adapted the config_get methods. It's much simpler now and should be easy.